contentcuration/contentcuration/frontend/accounts/pages/Create.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+
+  <ImmersiveModalLayout
+    previousPage="/"
+    :appBarText="$tr('title')"
+  >
+    <h1>{{ $tr('title') }}</h1>
+    <p>...</p>
+    <VBtn primary @click="submit">
+      Finish
```

Comment:
"Finish" should be in `$trs` as well.

Reply:
Same with the other English messages in the other Vue components' 'template' tags :+1:.

Reply:
yeah this was all stubbed out - the templates themselves are not ready